daemon/container_operations_windows.go

```diff
@@ -58,3 +43,7 @@ func isLinkable(child *container.Container) bool {
 func enableIPOnPredefinedNetwork() bool {
 	return true
 }
+
+func (daemon *Daemon) supportsHotAdd() bool {
```

Comment:
@mavenugo How about just dropping Hot and keeping it simple to isNetworkPluggable?